packages/playground/src/components/app-root/app-root.tsx

```diff
@@ -182,7 +182,7 @@ export class AppRoot {
 
     if (user.ethAddress) {
       provider.removeAllListeners(user.ethAddress);
```

Comment:
@snario If you're commenting the `provider.on()` line, this one doesn't make too much sense on its own.

Reply:
I think I uncommented that line